### ISSUE TITLE
Feature request - notify and log failed matches

### ISSUE THREAD

**Issue by RickDB** [RickDB/PlexAniSync#62]
If failed series matches are found:

- Log to file (failed_matches.txt)
- Push notification via Pushover / Pushbullet

For the Push notification part we need more control as to not send them on each run, perhaps local list to store send notifications.

Original request: [https://anilist.co/forum/thread/6443/comment/782721](https://anilist.co/forum/thread/6443/comment/782721)

**Comment by reconman** [comment]
So basically send the content of the file as notification?

**Comment by RickDB** [comment]
Yeah that would work with some little markup for readability sake :)

**Comment by KaiserBh** [comment]
#65 opened a PR.
